components/04-molecules/visual-elements/slider/_slider.scss

```diff
@@ -0,0 +1,88 @@
+.slider {
+  display: block;
+  overflow-x: scroll;
+  scroll-snap-type: x mandatory;
+}
+
+.slider__all {
+  @extend %list-unstyled;
+
+  position: relative;
+  display: flex;
+  margin: 0;
+}
+
+.slider__slide {
+  display: block;
+  flex: 0 0 100%;
+  scroll-snap-align: center;
+}
+
+.slider__page {
+  border-width: 1px;
+  border-color: $brand-muted;
+  border-right-style: solid;
+  margin-bottom: 0;
+}
+
+.slider:focus {
+  outline: 4px solid $brand-secondary-75;
+}
+
+.page__number__box {
+  position: absolute;
+  top: 4%;
+  width: 10%;
+  margin-right: 3%;
+  margin-left: 90%;
+}
+
+.page__number {
+  @extend %heading-font-regular;
+
+  display: inline-block;
+  padding: $spacer-xxs;
+  background: rgba(247, 247, 247, 0.7);
+  font-size: $font-size-xxs;
+}
+
+.instructions--hover,
+.instructions--focus,
+.instructions--hover--and--focus,
```

Comment:
.instructions--hover--and--focus Diese Klasse würde ich umbenennen in .instructions--hover-and-focus

Reply:
Hab ich so gemacht, hast völlig Recht :) Und das betrifft auch "instructions--touch--active".